components/styles.js

```diff
@@ -73,8 +73,8 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
-    left: 13,
-    top: 13,
+    left: (DEVICE_WIDTH / 14) - 14,
```

Comment:
Since the selectedDayCircle has absolute positioning, it was originally set to left: 13 and top: 13, because it happened to work on the emulator size I was working on at the time. When you switch to a different phone dimension, the circle was off center, so this ensures it centers the day text + circle on the day button. Each button has width is DEVICE_WIDTH / 7, so half of the button is DEVICE_WIDTH / 14. The selected day circle is set at height = width = 28, so setting the absolute left = top = DEVICE_WIDTH/14 - 28/2 is ensures the day circle is centered